routes/common/common.js

```diff
@@ -397,3 +400,76 @@ exports.batchPushDistricts = function(req, res) {
         }
     })
 };
+/**
+ * Generates the dashboard performance metrics from the database
+ * by counting the number of calls, demands or transactions
+ * made within a period of time.
+ * @param  {[type]} req [description]
+ * @param  {[type]} res [description]
+ */
+exports.getStats = function(req, res){
```

Comment:
@paradoxxjm gosh we need to put in a test suite on this... hard to go through this in the code review :(  
I will get this done on a weekend.